src/databasetasks.cpp

```diff
@@ -9,12 +9,6 @@
 
 extern Dispatcher g_dispatcher;
 
-void DatabaseTasks::start()
-{
-	db.connect();
-	ThreadHolder::start();
-}
-
 void DatabaseTasks::threadMain()
```

Comment:
Now we will have only one singleton for the `Database`. Is there an issue with database_lock?